js/stex.js

```diff
@@ -1171,7 +1171,7 @@ module.exports = class stex extends Exchange {
                 result['fee'] = undefined;
             }
         }
-        return result;
+        return this.safeOrder (result, market);
```

Comment:
If we are converting to safeOrder here, then we should pass strings instead of numbers to it, including the fees, so line 1160 has to be changed from 
```JavaScript
const feeCost = this.safeNumber (fees[i], 'amount');
```
to 
```JavaScript
const feeCost = this.safeString (fees[i], 'amount');
```